VKApp/VKApp/Controllers/LoginViewController.swift

```diff
@@ -0,0 +1,163 @@
+        static let whiteColorName = "WhiteColor"
+        static let loginPlaceholderText = "Email или телефон"
+        static let passwordPlaceholderText = "Пароль"
+        static let enterButtonText = "Войти"
+        static let forgotPasswordButtonText = "Забыли пароль?"
+        static let loginSegueIdentifier = "LoginSegue"
+        static let titleAlertText = "Ошибка"
+        static let messageAlertText = "Логин или пароль неверные"
+        static let correctLoginText = "admin"
+        static let correctPasswordText = "12345"
+        static let okText = "OK"
+    }
+
+    // MARK: - Private Outlets
+
+    @IBOutlet private var loginTextField: UITextField!
+    @IBOutlet private var passwordTextField: UITextField!
+    @IBOutlet private var enterButton: UIButton!
+    @IBOutlet private var forgotPasswordButton: UIButton!
+    @IBOutlet private var mainScrollView: UIScrollView!
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillShown(notification:)),
+            name: UIResponder.keyboardWillShowNotification,
+            object: nil
+        )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillHide(notification:)),
+            name: UIResponder.keyboardWillHideNotification,
+            object: nil
+        )
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(hideKeyboard))
+        mainScrollView.addGestureRecognizer(tapGesture)
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+    }
+
+    @IBAction func loginButtonAction(_ sender: UIButton) {
+        if checkLoginInfo() {
+            performSegue(withIdentifier: Constants.loginSegueIdentifier, sender: self)
+        } else {
+            showAlert(title: Constants.titleAlertText, message: Constants.messageAlertText)
+        }
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        setupMainView()
+        setupLoginTextField()
+        setupPasswordTextField()
+        setupEnterButton()
+        setupForgotPasswordButton()
+    }
+
+    private func setupMainView() {
+        view.backgroundColor = UIColor(named: Constants.lightBlueColorName)
+    }
+
+    private func setupLoginTextField() {
+        loginTextField.delegate = self
+        loginTextField.placeholder = Constants.loginPlaceholderText
+        loginTextField.font = UIFont.systemFont(ofSize: 18)
```

Comment:
```suggestion
        loginTextField.font = UIFont.systemFont(ofSize:)
```
размер шрифта везде одинаков -> в константы